packages/compass-e2e-tests/package.json

```diff
@@ -22,7 +22,9 @@
     "insert-data": "ts-node ./scripts/insert-data.ts",
     "unzip-fixtures": "ts-node ./scripts/gunzip.ts fixtures/*.gz",
     "testslow": "env DEBUG=hadron*,mongo*,compass* npm run test -- --disable-start-stop --bail",
-    "testfast": "env DEBUG=hadron*,mongo*,compass* npm run test -- --no-native-modules --no-compile --disable-start-stop --bail"
+    "testfast": "env DEBUG=hadron*,mongo*,compass* npm run test -- --no-native-modules --no-compile --disable-start-stop --bail",
+    "coverage-merge": "nyc merge .log/coverage .nyc_output/coverage.json",
+    "coverage-report": "npm run coverage-merge && nyc report --reporter=lcov --cwd ../.. --temp-dir ./packages/compass-e2e-tests/.nyc_output --report-dir ./packages/compass-e2e-tests/coverage"
```

Comment:
I had to set --cwd to make it take in all the packages, but then once you set that you have to set --temp-dir and --report-dir as well because those are interpreted relative to --cwd as well.

Reply:
Tiny nit / suggestion: maybe makes sense to use a [config file](https://github.com/istanbuljs/nyc#configuration-files) instead of inline args for ease of reading the options? And then you can even have these context comments there if you use yaml or js config format